webapp/app/config.py

```diff
@@ -95,6 +95,12 @@ class DevelopmentConfig(BaseConfig):
     SESSION_COOKIE_SECURE = False  # Because Safari can not send Secure Cookies via HTTP to localhost
 
     ERICA_BASE_URL = environ.get('ERICA_BASE_URL') or 'http://0.0.0.0:8000/01'
+    """ Use the following variables to make tests locally with the job queue dummy routes.
```

Comment:
@JulianRoesner 

Should we maybe create a new config for this? For example:

```
class EricaVersionUpgradeConfig(DevelopmentConfig):
    USE_MOCK_API = True
    MOCK_MIN_REQUEST_COUNT = 5
    MOCK_DELIVER_FAIL_ON_POST = False
    MOCK_DELIVER_FAIL_ON_GET = False
```

Are these configurations temporary or do we want to use this in the future? Maybe it also makes sense to configure the current and next versions so we don't use "01" & "02" in the code anymore?

```
class EricaVersionUpgradeConfig(DevelopmentConfig):
    USE_MOCK_API = True
    MOCK_MIN_REQUEST_COUNT = 5
    MOCK_DELIVER_FAIL_ON_POST = False
    MOCK_DELIVER_FAIL_ON_GET = False
    CURRENT_VERSION="01"
    NEXT_VERSION="02"
```

This would also mean that we no longer explicitly specify the version in the ERICA_BASE_URL='http://0.0.0.0:8000/' but store the version in a separate configuration ERICA_VERSION="01".

Wdyt?

Reply:
> Are these configurations temporary or do we want to use this in the future? Maybe it also makes sense to configure the current and next versions so we don't use "01" & "02" in the code anymore?

I agree with you. Victor and I also talked about this and we probably should change it in that direction. For me, this does not have to be part of this PR because this will also need some changes to the infrastructure and we need to think about how to deploy those.
- Use ERICA_URL & ERICA_VERSION
- Keep ERICA_BASE_URL the same
- Add ERICA_URL as a new secret to the environments and ERICA_VERSION to the code

Reply:
> Should we maybe create a new config for this? For example:

I agree. I would not want the default dev config to use the mocked Erica. What about:

```python
class MockedDevelopmentConfig(DevelopmentConfig):
    USE_MOCK_API = Ture
```

Imo the other options should be moved to the MockErica (s. other comment)

Reply:
Thats fine for me, we can change the ERICA_BASE_URL in another ticket / PR